01_02_Combat_Rules.txt

```diff
@@ -888,12 +888,31 @@ above. Even if you had 5 health and took 80 damage, when you go down you go to 0
 health. Any additional attacks you sustain will push you into negative health. 
 
 Stabilize:
-	As an action, a teammate may bring you back up to 1 health point by succeeding 
-on a DC 80 Medicine skill check to stabilize you. Medicine skill, INT mod, and any
-bonuses from medical equipment such as medic-kits are added to the roll. Once stabilized,
-you may take actions  and speak as normal.
-
-	Don't forget that a recently stabilized character may need  to take actions to
+	As an action, a character may choose an adjacent, Downed character and 
+attempt to Stabilize them. That character makes DC 80 Dex-based skill check: 
+Medicine for primarily organic targets, or Repair for primarily mechanical targets.
+If successful, the target is revived and now has 1 HP, is no longer Downed, and 
+may take Actions as normal within the initiative order. A character may be revived 
+in a single encounter a number of times equal to half their Fort, minimum two. Past 
+this limit, stabilized characters are unconscious in addition to the other effects 
+of Stabilization. Note that most persistent damage effects, like bleeds or burns, 
+fall off of characters when they are Downed.
+
+	If a character fails a stabilization check, nothing happens. It's just 
+taking longer to stem the bleeding. On critical success (91-100 before modifiers), 
+your excellent care allows you to choose whether the target counts this revival 
+against their revivals per encounter limit. On critical failure (01-05 before 
+modifiers), the bleeding (or equivalent) gets worse: the target must immediately 
+roll an additional Death Save. If the total check is then still high enough to 
```

Comment:
It's a little confusing to me that you could succeed at a task when you critically failed the roll.

Reply:
I was thinking success-at-a-cost style for this action, because I didn't think I would like ever auto-failing a non-Doctor serious enough to put 50+ skill points into Medicine (/non-Engineer with 50 points in Repair). Not unwilling to revert.

Reply:
@bleehu your thoughts on this also solicited